### PR TITLE
proper t-coffee error message if global alignment fails

### PR DESCRIPTION
**Pull request by joergi-w**
see issue #2067

### CODE REVIEW

**Comment by h-2** [comment]
Hm, shouldn't you have a global exception handler somewehere that also catches other bad_allocs?

File: include/seqan/graph_msa/graph_align_tcoffee_library.h [hunk 1]
```diff
@@ -995,7 +995,17 @@ appendSegmentMatches(StringSet<TString, Dependent<TSpec> > const& str,
...
-        TScoreValue myScore = globalAlignment(matches, pairSet, score_type, ac, Gotoh());
+        TScoreValue myScore{};
+        try {
+            myScore = globalAlignment(matches, pairSet, score_type, ac, Gotoh());
+        }
+        catch (const std::bad_alloc & exception)
+        {
+            std::cerr << "Allocation for globalAlignment failed. Use smaller data or try seeded alignment. \n"
+                      << exception.what() << std::endl;
+            exit(1);
```
Comment (rrahn) [inline include/seqan/graph_msa/graph_align_tcoffee_library.h]: please use exit status: http://en.cppreference.com/w/cpp/utility/program/EXIT_status

**Comment by joergi-w** [comment]
Thanks for the advice! I put the exception handler to a higher level, so it catches bad_allocs also for different program settings. And I use the EXIT_FAILURE macro if something goes wrong.

**Comment by h-2** [comment]
rebuild

**Pull request merged**